doc/cephfs/nfs.rst

```diff
@@ -79,3 +79,219 @@ Current limitations
 
 - Per running ganesha daemon, FSAL_CEPH can only export one Ceph file system
   although multiple directories in a Ceph file system may be exported.
+
+Exporting over NFS clusters deployed using rook
+===============================================
+
+In this tutorial we will be using minikube for setting up a single node test
+cluster. If you already have a kubernetes cluster deployed, please directly
+start with rook part.
+
+.. note:: Configuration of this tutorial should not be used in a a real
+          production cluster. For the purpose of simplification, the security
+          aspects of Ceph are overlooked in this setup.
+
+Minikube Setup
+--------------
+
+Installation::
+
+        dnf install kubernetes-client docker-common git qemu-kvm
+        cd /tmp
+        curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube
+        curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2 && chmod +x docker-machine-driver-kvm2
+        cp minikube docker-machine-driver-kvm2 /usr/local/bin
+
+.. note:: Place the minkube and docker executables in $PATH. Make sure $HOME has enough space as minikube requires about 25 GB.
+
+
+Deploy Minikube Cluster::
+
+        minikube start --vm-driver=kvm2
+
+.. note:: Add --memory and --cpus options to provide custom RAM and CPU values for the VM.
+
+List the running pods::
+
+        kubectl get pods --all-namespaces
+
+Rook Setup And Cluster Deployment
+---------------------------------
+
+Clone the rook repository::
+
+        git clone https://github.com/rook/rook.git
+
+Deploy the Rook Operator::
+
+        cd cluster/examples/kubernetes/ceph
+        kubectl create -f common.yaml
+        kubectl create -f operator.yaml
+
+.. note:: Nautilus release or latest Ceph image should be used.
+
+Before proceding check if the pods are running::
+
+        kubectl -n rook-ceph get pod
+
+.. note:: For any troubleshooting on any pod use::
+          kubectl describe -n rook-ceph pod <pod-name>
+
+In cluster-test.yaml file set **allowMultiplePerNode** to **true** under mon. As we
+will be using 3 monitors. If using minkube cluster change the **dataDirHostPath**
+to **/data/rook**. This is to make sure data persists across reboots.
+
+Deploy the ceph cluster::
+
+        kubectl create -f cluster-test.yaml
+
+Now check if you have one Ceph monitor, manager and OSD running.
```

Comment:
Are you suggesting that the user now check their daemons? If so, I'd suggest giving a quick kubectl exec command line to do that. If you're indicating that they're going to check that a couple steps down, when they first run ceph -s, then you can probably cut this line.